ext/core.package.json

```diff
@@ -0,0 +1 @@
+../core/package.json
```

Comment:
What's this file doing? I've not seen a package.json with just a path in before.

Reply:
It's a symlink, not a file with a single line. The point is to enable Grist Desktop to read the package.json from grist-core to retrieve versioning info.

If you ask why this symlink is needed, it has something to do with our chaotic directory structure that makes editors and tsc resolve imports differently.

Reply:
Ahhhhhh - okay, didn't realise this is how symlinks show in Git. Awesome, happy to leave this for now then as a workaround :D